src/Symfony/Component/HttpKernel/Event/GetResponseForExceptionEvent.php

```diff
@@ -36,6 +36,11 @@ class GetResponseForExceptionEvent extends GetResponseEvent
      */
     private $exception;
 
+    /**
+     * @var bool
+     */
+    private $allowSuccessfulResponse = false;
```

Comment:
Good point - I used this as it was proposed in the original issue, but I'll re-name it to something more appropriate.